atomate/vasp/fireworks/lobster.py

```diff
@@ -0,0 +1,118 @@
+# coding: utf-8
+"""
+This module defines fireworks  that can be used to handle Lobster calculations that are based on VASP wavefunctions.
+"""
+
+import logging
+import os
+from typing import List, Union
+
+from atomate.common.firetasks.glue_tasks import DeleteFiles, PassCalcLocs, DeleteFilesPrevFolder
+from atomate.vasp.config import DB_FILE
+from atomate.vasp.firetasks.glue_tasks import CopyVaspOutputs
+from atomate.vasp.firetasks.lobster_tasks import WriteLobsterinputfromIO, RunLobster, LobsterRunToDb
+from custodian.custodian import ErrorHandler, Validator
+from fireworks import Firework
+from pymatgen.core.structure import Structure
+
+__author__ = "Janine George, Guido Petretto"
+__email__ = 'janine.george@uclouvain.be, guido.petretto@uclouvain.be'
+
+LOBSTER_CMD = ">>lobster_cmd<<"
+VASP_OUTPUT_FILES_without_PPKI = ["OUTCAR", "vasprun.xml", "CHG", "CHGCAR", "CONTCAR", "DOSCAR", "EIGENVAL", "IBZKPT",

```

Comment:
See my previous note about lists of VASP files.

Reply:
will change this.